### PR TITLE
Test PR

### PR DESCRIPTION
**Pull request by maleadt**
Testing new PkgEval functionality.

@nanosoldier `runtests(["GPUArraysCore"])`

### CODE REVIEW

**Comment by nanosoldier** [comment]
Update on PkgEvalJob JuliaGPU/Adapt.jl@be00706 vs. JuliaGPU/Adapt.jl@8f009b2: Accepted

**Comment by nanosoldier** [comment]
[Your package evaluation job](https://github.com/JuliaGPU/Adapt.jl/pull/55#issuecomment-1376904767) has completed - no new issues were detected.
A full report can be found [here](https://github.com/maleadt/NanosoldierReports/blob/master/pkgeval/by_hash/be00706_vs_8f009b2/report.md).

**Comment by maleadt** [comment]
@nanosoldier `runtests(["GPUArraysCore"])`

**Comment by nanosoldier** [comment]
Update on PkgEvalJob JuliaGPU/Adapt.jl@be00706 vs. JuliaGPU/Adapt.jl@8f009b2: Accepted

**Comment by nanosoldier** [comment]
Update on PkgEvalJob JuliaGPU/Adapt.jl@be00706 vs. JuliaGPU/Adapt.jl@8f009b2: Running

**Comment by nanosoldier** [comment]
[Your package evaluation job](https://github.com/JuliaGPU/Adapt.jl/pull/55#issuecomment-1376907224) has completed - no new issues were detected.
A full report can be found [here](https://github.com/maleadt/NanosoldierReports/blob/master/pkgeval/by_hash/be00706_vs_8f009b2/report.md).

**Comment by maleadt** [comment]
@nanosoldier `runtests(["GPUArraysCore"])`

**Comment by nanosoldier** [comment]
Update on PkgEvalJob JuliaGPU/Adapt.jl@be00706 vs. JuliaGPU/Adapt.jl@8f009b2: Accepted

**Comment by nanosoldier** [comment]
Update on PkgEvalJob JuliaGPU/Adapt.jl@be00706 vs. JuliaGPU/Adapt.jl@8f009b2: Running

**Comment by nanosoldier** [comment]
[Your package evaluation job](https://github.com/JuliaGPU/Adapt.jl/pull/55#issuecomment-1376910654) has completed - no new issues were detected.
A full report can be found [here](https://github.com/maleadt/NanosoldierReports/blob/master/pkgeval/by_hash/be00706_vs_8f009b2/report.md).

**Comment by maleadt** [comment]
@nanosoldier `runtests(["GPUArraysCore"])`

**Comment by maleadt** [comment]
@nanosoldier `runtests(["GPUArraysCore"])`

**Comment by nanosoldier** [comment]
Update on PkgEvalJob maleadt/Adapt.jl@be00706 vs. JuliaGPU/Adapt.jl@8f009b2: Accepted

**Comment by nanosoldier** [comment]
Update on PkgEvalJob maleadt/Adapt.jl@be00706 vs. JuliaGPU/Adapt.jl@8f009b2: Running

File: Project.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name = "Adapt"
 uuid = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
-version = "3.4.0"
+version = "3.4.1"
```
Comment (maleadt) [inline Project.toml:3]: @nanosoldier `runtests(["GPUArraysCore"])`

**Comment by nanosoldier** [comment]
Update on PkgEvalJob maleadt/Adapt.jl@be00706: Accepted

**Comment by nanosoldier** [comment]
Update on PkgEvalJob maleadt/Adapt.jl@be00706: Running

**Comment by nanosoldier** [comment]
[Your package evaluation job](https://github.com/JuliaGPU/Adapt.jl/pull/55#discussion_r1065491656) has completed - no issues were detected.
A full report can be found [here](https://github.com/maleadt/NanosoldierReports/blob/master/pkgeval/by_hash/be00706/report.md).

**Pull request closed without merging**